platformio.ini

```
@@ -35,7 +35,7 @@ monitor_port    =
 build_type      = release
 upload_speed    = 1500000
 build_flags     = -std=gnu++2a
-                  -Dregister=                       ; Sinister: redefine 'register' so FastLED can use that keyword under C++17 and later
+                  -Dregister=                       ; Sinister: redefine 'register' so FastLED can use that keyword under C++17
```

I think the "...and later" is more accurate. This could make it even clearer:

```suggestion
                  -Dregister=                       ; Sinister: redefine 'register' so FastLED can use that keyword under C++17 and later C++ versions
```